common/src/main/java/io/druid/math/expr/ExprEval.java

```diff
@@ -150,7 +150,7 @@ public final double asDouble()
   {
     private DoubleExprEval(Number value)
     {
-      super(value);
+      super(Preconditions.checkNotNull(value, "value"));
```

Comment:
I see. Then, would you remove ```@Nullable``` annotations at [DoubleExpr.getLiteralValue()](https://github.com/druid-io/druid/blob/master/common/src/main/java/io/druid/math/expr/Expr.java#L155) and [LongExpr.getLiteralValue()](https://github.com/druid-io/druid/blob/master/common/src/main/java/io/druid/math/expr/Expr.java#L95)?